go/src/koding/klient/machine/index/index.go

```diff
@@ -13,20 +13,20 @@ import (
 	"runtime"
 	"sort"
 	"sync"
+	"sync/atomic"
 
 	"github.com/djherbis/times"
 )
 
 // Version stores current version of index.
 const Version = 1
 
-// EntryMeta describes entry metadata.
-type EntryMeta int32
```

Comment:
I found no use for a separate type, except it requires a lot of casting. Thus I simplified the api.